src/i_richpresence.c

```diff
@@ -0,0 +1,109 @@
+//
+//  Copyright (C) 2025 Roman Fomin
+//
+//  This program is free software; you can redistribute it and/or
+//  modify it under the terms of the GNU General Public License
+//  as published by the Free Software Foundation; either version 2
+//  of the License, or (at your option) any later version.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+
+#include "config.h"
+#include "doomtype.h"
+#include "i_printf.h"
+
+#ifdef HAVE_DISCORD_RPC
+
+#include <time.h>
+
+#include "discord_rpc.h"
+
+#define DEFAULT_DISCORD_APP_ID "1425007665855336460"
```

Comment:
Huh, where does this come from?

Reply:
I registered "Woof!" app in Discord.

Reply:
Cool! Is there something like a link to it?

Reply:
I don't know. Check out Discord client, I'm running woof.

Reply:
What a fun! 😁